.travis.yml

```diff
@@ -14,11 +14,14 @@
 
 language: rust
 rust:
-  - 1.31.0
```

Comment:
Please don't remove this. We are testing this version to know if we ever break builds on older Rust.

Reply:
Reverted.